src/pages/editor/PublishingStrategyOverview.tsx

```diff
@@ -0,0 +1,147 @@
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import RemoveCircleIcon from '@mui/icons-material/RemoveCircle';
+import { Box, Divider, Link, styled, Typography } from '@mui/material';
+import { Helmet } from 'react-helmet-async';
+import { Trans, useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { PageSpinner } from '../../components/PageSpinner';
+import { RootState } from '../../redux/store';
+import { CustomerRrsType } from '../../types/customerInstitution.types';
+
+const PublishStrategyContainer = styled('div')({
+  padding: '0.5rem',
+  borderRadius: '0.5rem',
+  backgroundColor: 'white',
+  boxShadow: '0px 3px 3px 0px rgba(0, 0, 0, 0.30)',
+});
+
+export const StyledAccessRight = styled('div')({
+  display: 'flex',
+  gap: '0.5rem',
+});
+
+const coalitionSUrl = 'https://www.coalition-s.org/resources/rights-retention-strategy/';
+
+export const StyledAccessRightsContainer = styled('div')({
+  display: 'flex',
+  justifyContent: 'space-evenly',
+  marginTop: '0.5rem',
+  marginBottom: '0.5rem',
+});
+
+export const PublishingStrategyOverview = () => {
+  const { t } = useTranslation();
+  const customer = useSelector((store: RootState) => store.customer);
+
+  const currentPublishStrategy = customer?.publicationWorkflow;
+  const isRrs = customer?.rightsRetentionStrategy?.type === CustomerRrsType.RightsRetentionStrategy;
+  const isOverridableRrs =
+    customer?.rightsRetentionStrategy?.type === CustomerRrsType.OverridableRightsRetentionStrategy;
+
+  return (
+    <>
+      <Helmet>
+        <title id="publish-strategy-label">{t('editor.publish_strategy.publish_strategy')}</title>
```

Comment:
ser ikke ut til å brukes til noe?
```suggestion
        <title>{t('editor.publish_strategy.publish_strategy')}</title>
```